cumulus_library_data_metrics/us_core_v4/diagnosticreport_lab_must_support.jinja

```diff
@@ -2,9 +2,6 @@
 -- There is another non-lab-report-specific profile,
 -- but this is the DiagnosticReport profile that covers the LAB rows.
 
--- CUMULUS-QUIRK
--- Doesn't look for "presentedForm", because Cumulus strips that field.
```

Comment:
This was a copy-and-paste comment in error, afaict. DxReport Lab doesn't mark this field as must-support (it does mention it, but only in passing).